services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    else:
+        queryset = queryset.all()
+    return queryset
+
+
+def get_movie_by_id(
+    movie_id: int
+) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+    movie_title: str,
+    movie_description: str,
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> None:
```

Comment:
The return type annotation `-> None` is incorrect because the function actually returns a `Movie` object. Please update the return type annotation to `-> Movie`.